lmdeploy/pytorch/engine/engine_checker.py

```diff
@@ -89,3 +89,10 @@ def check(self):
             self.log_and_exit(mod_name='Engine',
                               message='max_batch_size should be'
                               f' greater than 0, but got {engine_config.max_batch_size}')
+
+        num_gpu_blocks = engine_config.num_gpu_blocks
+        if num_gpu_blocks > 0 and num_gpu_blocks < 16:
```

Comment:
This appears to be more of a workaround than a root cause fix.  We might need a more permanent solution to address the root cause later.